### ISSUE TITLE
GTFS.org UI items

### ISSUE THREAD

**Issue by Alessandro100** [MobilityData/gtfs.org#512]
The launch of gtfs.org is ~August 20, there are corrections to the UI that need to be done before release

Style / functionality touch ups
- Landing
    - Some images have boarders when they shouldn’t black
    - Some icons have a black boarder when they shouldnt
    - Translation styling
    - Animations on landing
    - Mobile

- Buttons with external pop outs (other pages)
    - Check functionality

**Comment by fredericsimard** [comment]
@Alessandro100 I think this is no longer relevant, or am I wrong?

**Comment by Alessandro100** [comment]
yeah we can close this
